packages/dom/src/stripHTML.js

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-const buffer = document.createElement('div');
+const buffer =
+  typeof document !== 'undefined' ? document.createElement('div') : {};
 
 export default function stripHTML(string) {
```

Comment:
Do we need a `stripHTML` function that actually works on the server?

Reply:
I think we do. Probably will have to use some external library/package for it.

Reply:
There are packages available like [himalaya](https://www.npmjs.com/package/himalaya) ( 375kB ) but not very lightweight.

Reply:
Wouldn't it need something more like [string-strip-html](https://www.npmjs.com/package/string-strip-html)?

Still quite large though, so as long as we don't need it, we can postpone this.